### ISSUE TITLE
Add LDC datasets

### ISSUE THREAD

**Issue by mjpost** [mjpost/sacreBLEU#2]
If $LDC were defined, LDC datasets installed locally could be extracted in similar fashion without violating any licenses. However this would require writing an XML parser to handle multiple references since Python 3 doesn't include one.

**Comment by ozancaglayan** [comment]
There's this standard library: https://docs.python.org/3/library/xml.etree.elementtree.html

**Comment by mjpost** [comment]
Yeah, I don't know why I thought we'd have to write one. For the Anthology we use [lxml](https://lxml.de).

In my experience the problem with many "community-wrapped" XML files (e.g., WMT) is that they do not parse.
